### PR TITLE
[PPML] Add command for SGX driver

### PR DESCRIPTION
**Pull request by ManfeiBai**

### CODE REVIEW

File: ppml/README.md [hunk 1]
```diff
@@ -1 +1,13 @@
 ## PPML (Privacy Preserving Machine Learning) 
+### Create SGX driver
+sudo apt install dkms
```
Comment (qiyuangong) [inline ppml/README.md:3]: Using ``` to format code of command block

Comment (ManfeiBai) [reply ppml/README.md:4]: > Using ``` to format code of command block

Thanks, done~

Comment (qiyuangong) [reply ppml/README.md:4]: Will this code block become a script?

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**